rpm/cray/csm/sle-15sp4/index.yaml

```diff
@@ -23,29 +23,37 @@
 #
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp4/:
   rpms:
-    - bos-reporter-2.5.5-1.noarch
-    - canu-1.7.4-1.x86_64
     - cf-ca-cert-config-framework-2.6.0-1.noarch
     - cfs-debugger-1.4.0-1.noarch
     - cfs-state-reporter-1.9.3-1.noarch
-    - cfs-trust-1.6.6-1.noarch
+    - cfs-trust-1.6.7-1.noarch
     - cray-cmstools-crayctldeploy-1.12.0-1.x86_64
     - cray-site-init-1.32.0-1.x86_64
-    - craycli-0.82.8-1.x86_64
-    - csm-node-identity-1.0.20-1.noarch
     - csm-ssh-keys-1.5.4-1.noarch
     - csm-ssh-keys-roles-1.5.4-1.noarch
-    - csm-testing-1.16.50-1.noarch
-    - goss-servers-1.16.50-1.noarch
     - hpe-csm-goss-package-0.3.21-hpe3.x86_64
-    - hpe-csm-scripts-0.5.6-1.noarch
+    - hpe-yq-4.33.3-1.aarch64
     - hpe-yq-4.33.3-1.x86_64
-    - ilorest-4.2.0.0-20.x86_64
     - iuf-cli-1.5.3-1.x86_64
     - libcsm-0.0.4-1.noarch
-    - loftsman-1.2.0-2.x86_64
-    - manifestgen-1.3.9-1.x86_64
+    - loftsman-1.2.0-3.x86_64
     - metal-nexus-1.2.3-1.x86_64
     - metal-observability-1.0.9-1.x86_64
+
+# Add some noos RPMs into this SLE repo to avoid breaking recipes which may
+# expect them to be found here
+https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/noos/:
+  rpms:
+    - bos-reporter-2.6.0-1.noarch
+    - canu-1.7.5-1.x86_64
+    - craycli-0.82.9-1.aarch64
+    - craycli-0.82.9-1.x86_64
+    - csm-node-identity-1.0.22-1.noarch
+    - csm-testing-1.16.51-1.noarch
+    - goss-servers-1.16.51-1.noarch
+    - hpe-csm-scripts-0.5.7-1.noarch
+    - ilorest-4.2.0.0-20.x86_64
```

Comment:
```suggestion
```

Reply:
These shouldn't need coverage, they'll be in the noos repo in nexus and are ours.. shouldn't be in any one else's recipe.